geometry_msgs/PoseStamped goal
std_msgs/Float64 time
---
bool success
